### PR TITLE
Keep dark theme for ListView header in RTL

### PR DESCRIPTION
**Pull request by OmrSi**
I've been at it all day, but something is very wrong with the listview header in RTL, you can't use `e. Graphics.DrawString` or `e. Graphics.FillRectangle`, they just don't work. The only thing that works is `TextRenderer.DrawText` for some reason.

This is how it is:
image

It's the best result I could get.

### CODE REVIEW

File: src/ui/Logic/DarkTheme.cs [hunk 1]
```diff
@@ -395,16 +395,17 @@ private static void ListView_DrawSubItem(object sender, DrawListViewSubItemEvent
...
 
-            e.DrawDefault = false;
-            using (var b = new SolidBrush(Color.FromArgb(Math.Max(BackColor.R - 9, 0), Math.Max(BackColor.G - 9, 0), Math.Max(BackColor.B - 9, 0))))
+            using (var slightlyDarkerBrush = new SolidBrush(Color.FromArgb(Math.Max(BackColor.R - 9, 0), Math.Max(BackColor.G - 9, 0), Math.Max(BackColor.B - 9, 0))))
             {
-                e.Graphics.FillRectangle(b, e.Bounds);
+                e.Graphics.FillRectangle(slightlyDarkerBrush, e.Bounds);
             }
 
             var strFormat = new StringFormat();
```
Comment (niksedk) [inline src/ui/Logic/DarkTheme.cs:411]: Is StringFormat disposable?

Comment (OmrSi) [reply src/ui/Logic/DarkTheme.cs:411]: Yep, it implements IDisposable.

**Comment by OmrSi** [comment]
@niksedk Is it true that using TextRenderer which works with GDI is better than Graphics.DrawString which works with GDI+?

**Comment by niksedk** [comment]
> 
> 
> @niksedk Is it true that using TextRenderer which works with GDI is better than Graphics.DrawString which works with GDI+?

Sorry, I don't know...

**Pull request merged**